src/security/cert_generators/file/certificate_db.cc

```diff
@@ -9,6 +9,8 @@
 #include "squid.h"
 #include "base/HardFun.h"
 #include "base/TextException.h"
+#include "compat/socket.h"
```

Comment:
API provided by this include is not used by this .cc file.
```suggestion
```

Reply:
removed . That file is not building anyway due to LDAP API mismatch